.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,27 @@
+<!--
 Thanks for taking the time for reporting an issue!
 
 Before reporting an issue on dd-trace-py, please be sure to provide all
 necessary information.
 
 If you're hitting a bug, make sure that you're using the latest version of this
 library.
+-->
+
+### Summary of problem
 
 ### Which version of dd-trace-py are you using?
 
 ### Which version of pip are you using?
 
 _ddtrace requires pip>=18 to install one of our pre-built wheels_
 
-### Which version of the libraries are you using?
+### Which supported libraries and their versions are you using?
```

Comment:
does "supported" make sense? what if the user is requesting support for a new version or library

Reply:
Good point! Dropped the "supported".

```suggestion
### Which libraries and their versions are you using?
```